builder-24/builder.toml

```diff
@@ -33,6 +37,28 @@ version = "0.19.6"
     version = "3.1.1"
     optional = true
 
+[[order]]
+  [[order.group]]
+    id = "heroku/nodejs-engine"
+    version = "3.2.1"
+    optional = true
+  [[order.group]]
+    id = "heroku/nodejs-yarn"
+    version = "3.2.1"
+    optional = true
+  # TODO: Enable once `heroku/jvm` supports Heroku-24.
+  # [[order.group]]
+  #   id = "heroku/jvm"
+  #   version = "TBD"
+  #   optional = true
```

Comment:
fyi @schneems @Malax I've had to comment out the optional `heroku/jvm` entry in the Ruby detection group for now, since `heroku/jvm` doesn't yet support Heroku-24. Once it does, this can be uncommented as part of adding the JVM CNBs to this builder.